python/pacemaker/_cts/audits.py

```diff
@@ -287,13 +304,18 @@ def __call__(self):
         self._cm.ns.wait_for_all_nodes(self._cm.env["nodes"])
 
         for node in self._cm.env["nodes"]:
-            found = self._find_core_on_fs(node, "/var/lib/pacemaker/cores/*", "Pacemaker")
-            if found:
-                result = False
+            if self._cm.env["have_systemd"]:
+                found = self._find_core_with_coredumpctl(node)
+                if found:
```

Comment:
don't really need this `found` variable but it arguably makes clearer what the `_find*` functions are returning